whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/BouncedMessageParser.java

```diff
@@ -246,4 +251,10 @@ private static ContentType contentType(final String contentType) {
         }
     }
 
+    private static byte[] getMimeMessageBytes(final MimeMessage message) throws MessagingException, IOException {
```

Comment:
Move this method to the DAO where the MimeMessage is written to the database

Reply:
I will do this, but I think that we shouldn't add extra logic into a dao layer